src/App.css

```diff
@@ -6,145 +6,240 @@
+  padding-inline: 10px;
 }
 
 .logo-container {
-    display: flex;
-    align-items: center; /* Centering the logo image vertically */
+  display: flex;
+  align-items: center; /* Centering the logo image vertically */
 }
 
 .logo {
-    width: 50px;
-    height: 50px;
+  width: 50px;
+  height: 50px;
 }
 
 .nav-list {
-    display: flex; /* Making the nav-list appear horizontal */
+  display: flex; /* Making the nav-list appear horizontal */
 }
 
 .nav-list > li {
-    list-style: none;
-    margin-left: 20px;
+  list-style: none;
+  margin-left: 20px;
 }
 
-.nav-list > li > a, .mob-nav-list > li > a{
-    text-decoration: none;
-    color: #2f4f4f;
+.nav-list > li > a,
+.mob-nav-list > li > a {
+  text-decoration: none;
+  color: #2f4f4f;
 }
 
 .mobile-nav-button {
-    display: none;
+  display: none;
 }
 
 .mob-nav-list-container {
-    position: absolute;
-    top: 30px; 
-    width: 100%;
-    text-align: center;
-    z-index: 1;
+  position: absolute;
+  top: 30px;
+  width: 100%;
+  text-align: center;
+  z-index: 1;
 }
 
 .mob-nav-list {
-    padding: 0;
-    background-color: #fff; /* Adjust the background color as needed */
-    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.5);
-    width: 60%;
-    margin-inline: auto;
+  padding: 0;
+  background-color: #fff; /* Adjust the background color as needed */
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.5);
+  width: 60%;
+  margin-inline: auto;
 }
 
 .mob-nav-list > li {
-    list-style: none;
-    padding-block: 15px;
+  list-style: none;
+  padding-block: 15px;
 }
 
 .sheSharp {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 24px;
-    font-weight: 500;
-    color: #2f4f4f;
-    display: flex;
-    justify-content: center;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 24px;
+  font-weight: 500;
+  color: #2f4f4f;
+  display: flex;
+  justify-content: center;
 }
 
 .styledData {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 16px;
-    line-height: 26px;
-    font-weight: 400;
-    color: #2f4f4f;
-    display: flex;
-    justify-content: center;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 16px;
+  line-height: 26px;
+  font-weight: 400;
+  color: #2f4f4f;
+  display: flex;
+  justify-content: center;
 }
 
 .container {
-    border-radius: 20px;
-    border-color: #2f4f4f;
-    border-style: solid;
-    border-width: 1px;
-    margin: auto;
-    width: 30%;
-    box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
-    padding: 40px;
-    transition: transform 0.3s ease-in-out;
+  border-radius: 20px;
+  border-color: #2f4f4f;
+  border-style: solid;
+  border-width: 1px;
+  margin: auto;
+  width: 30%;
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
+  padding: 40px;
+  transition: transform 0.3s ease-in-out;
 }
 
 .container:hover {
-    transform: scale(1.1);
+  transform: scale(1.1);
+}
+
+/* FOOTER STYLES */
+
+.footer {
+  background-color: #2f4f4f;
```

Comment:
This background color is quite dark, perhaps we can go for something lighter more inline with our brand colors? Here's a reference:
<img width="974" alt="Screenshot 2023-10-20 at 18 06 51" src="https://github.com/shesharpnl/hackathon-2023.react-js.base-example/assets/78166655/d974805d-73b0-4bd9-9a16-be10dbcc5028">